internal/payload/forum.go

```diff
@@ -7,7 +7,8 @@ import (
 )
 
 type Forum struct {
-	ID          int    `json:"id,omitempty"`
+	GroupID     int    `json:"groupid,omitempty"`
```

Comment:
No need for this. Access to forum explicitly needs group ID to be specified, since it is a resource under the group.

i.e.

`/groups/{id}/forums`